nym-vpn-core/crates/nym-vpnc/src/main.rs

```diff
@@ -188,6 +189,14 @@ async fn store_account(client_type: ClientType, store_args: &cli::StoreAccountAr
     Ok(())
 }
 
+async fn refresh_account_state(client_type: ClientType) -> Result<()> {
+    let mut client = vpnd_client::get_client(client_type).await?;
+    let request = tonic::Request::new(RefreshAccountStateRequest {});
+    let response = client.refresh_account_state(request).await?.into_inner();
+    println!("{:#?}", response);
```

Comment:
I think that println! is some leftover from your debugging?

Reply:
This is the cli client, and it's pretty rudimentary at the moment so that part is intended :)